app/api/event_copyright.py

```diff
@@ -71,20 +73,20 @@ def before_create_object(self, data, view_kwargs):
 
 class EventCopyrightDetail(ResourceDetail):
 
-    def before_patch(self, args, kwargs):
+    def before_patch(self, args, kwargs, data):
```

Comment:
I think this has to be done in other APIs also where `before_patch` is used. Is it good to make an issue for this?